docs/standard/base-types/best-practices.md

```diff
@@ -79,123 +80,126 @@ The particular way in which you call regular expression matching methods can hav
 
-The following example compares the performance of compiled and interpreted regular expressions when reading the first ten sentences and when reading all the sentences in the text of Theodore Dreiser's *The Financier*. As the output from the example shows, when only ten calls are made to regular expression matching methods, an interpreted regular expression offers better performance than a compiled regular expression. However, a compiled regular expression offers better performance when a large number of calls (in this case, over 13,000) are made.
+The following example compares the performance of compiled and interpreted regular expressions when reading the first 10 sentences and when reading all the sentences in the text of Theodore Dreiser's *The Financier*. As the output from the example shows, when only 10 calls are made to regular expression matching methods, an interpreted regular expression offers better performance than a compiled regular expression. However, a compiled regular expression offers better performance when a large number of calls (in this case, over 13,000) is made.
 
 [!code-csharp[Conceptual.RegularExpressions.BestPractices#5](../../../samples/snippets/csharp/VS_Snippets_CLR/conceptual.regularexpressions.bestpractices/cs/compare1.cs#5)]
 [!code-vb[Conceptual.RegularExpressions.BestPractices#5](../../../samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.regularexpressions.bestpractices/vb/compare1.vb#5)]
 
-The regular expression pattern used in the example, `\b(\w+((\r?\n)|,?\s))*\w+[.?:;!]`, is defined as shown in the following table.
+The regular expression pattern used in the example, `\b(\w+((\r?\n)|,?\s))*\w+[.?:;!]`, is defined as shown in the following table:
 
 |Pattern|Description|
 |-------------|-----------------|
-|`\b`|Begin the match at a word boundary.|
-|`\w+`|Match one or more word characters.|
-|<code>(\r?\n)&#124;,?\s)</code>|Match either zero or one carriage return followed by a newline character, or zero or one comma followed by a white-space character.|
-|<code>(\w+((\r?\n)&#124;,?\s))*</code>|Match zero or more occurrences of one or more word characters that are followed either by zero or one carriage return and a newline character, or by zero or one comma followed by a white-space character.|
-|`\w+`|Match one or more word characters.|
-|`[.?:;!]`|Match a period, question mark, colon, semicolon, or exclamation point.|
+|`\b`|Begins the match at a word boundary.|
```

Comment:
```suggestion
|`\b`|Begin the match at a word boundary.|
```

Reply:
If you don't want it to be a command, then it should be something like "Specifies that the match is to begin at a word boundary".